### ISSUE TITLE
Players randomly lose all their levels

### ISSUE THREAD

**Issue by BlackBeltPanda** [mcMMO-Dev/mcMMO#2397]
No errors in the console; no clue why it happens, but players will randomly lose all of their McMMO levels.

Server version: 1.8-R0.1-SNAPSHOT Spigot
McMMO version: 1.5.02-SNAPSHOT-b21

**Comment by rudolfs001** [comment]
I am having a very similar issue.

I'm using 1.5.01-b8 (Tried 1.5.02-b22, but had the same issue).

I'm using flatfile

all but 8 or so players will be deleted from the flatfile. It's not that their stats are set to 0, but that they are simply deleted. The 8 players that don't get deleted are sometimes different, but sometimes the same. Their stats are where they should be.

I have repopulated the file multiple times from backups, and everything will be peachy for a while, then (I think on restarts), all but 8 players will be deleted. I get no errors in the console, and cannot find any configs to change that would be applicable. Also, I did not get this issue before upgrading to 1.5.01
